data/json/recipes/recipe_food.json

```
@@ -3234,7 +3234,7 @@
     "skill_used": "cooking",
     "difficulty": 4,
     "time": "12 m",
-    "autolearn": true,
+    "book_learn": [ [ "baking_book", 1 ] ],
```

```suggestion
    "book_learn": [ [ "baking_book", 1 ], [ "cookbook_daintydishes", 6 ] ],
```
We should probably have another "for kids" cookbook with stuff like cookies and some other basics

---

I've added a childrens cookbook but haven't added recipes to it yet.